### ISSUE TITLE
mailcow-dockerized - HDD fail. Rebuild from backup

### ISSUE THREAD

**Issue by lukaspavelka** [mailcow/mailcow-dockerized#5425]
### Summary

Hello everyone. 

Tonight i have issue on home server and my hdd died. I have backuped directory where was mailcow-dockerized like this: 

![image](https://github.com/docker/compose/assets/48788338/f2c71011-28c7-4f66-a7b1-93060a6e29cb)

What should i do to build again enviroment and have all data like it was before crash of HDD?

I try:
![image](https://github.com/docker/compose/assets/48788338/be5922e6-1cab-474f-9f44-608f50279440)

i also try check .env file inside folder and there is only one line: mailcow.conf

### Motivation

What sould i do ? Please can someone be my hero ? :) ... 🥇 

### Additional context

Thanks for all support.

**Comment by bundyland** [comment]
It looks like you have missed to backup the docker folders (`/var/lib/docker`) as here are the most important files (like all mails, crypt-keys etc.).

In the future you should better use the backup and restore script which is made to have a real full backup. Have a look under `/your_path_to/mailcow-dockerized/helper-scripts` or here: [https://docs.mailcow.email/backup_restore/b_n_r-backup/](url)

**Comment by lukaspavelka** [comment]
So i should install all from scratch ?

**Comment by bundyland** [comment]
Yes, this is what I would suggest. There is no other possibility if you don't have a proper backup of the docker folders.
And in future, better always create a complete backup with the corresponding script.

**Comment by lukaspavelka** [comment]
Thank you for highlighting the significance of ensuring proper backup of Docker folders. I completely agree with the need to create a complete backup using the corresponding script. It's essential to have a fail-safe solution in case of any unforeseen issues.

**Comment by lukaspavelka** [comment]
Btw i delete everything from folder where i have backup of mailcow-dockerized, than i did clean install and now all containers working  all looks good, website for login is here, but when i put default admin and pass i cant log in. when i try to log in with bad pass i get answer password is not good... 

i also try reset pass and after reset also no luck. After i click on login button i can get in, just redirected to
: https://192.168.8.62:2443/debug
Blank website with no text...

i also try domain dns and website working but also cant log in. 

pass is 100% good

**Comment by patschi** [comment]
As this is a old "issue" and not technically a bug in mailcow, closing this.
